docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -238,6 +243,20 @@ sometimes resolved by additionally installing the ``apptainer-setuid`` package.
 -  Enroot does not provide a mechanism for sharing containers. Each user must create any containers
    needed by their Determined experiments prior to creating the experiment.
 
+-  Some Docker features do not have an exact replacement in Enroot.
```

Comment:
The features listed below are Docker features?  Aren't they Determined features, not Docker?

Reply:
They are Determined features that support the use of Docker features that are not in Enroot.

This is the same wording we used above for Singularity.

I mean we could say `Some Docker features do not have an exact replacement in Enroot and therefore the associated Determined features are not supported`?